### PR TITLE
feat: add support for one dtype for all columns

### LINKED ISSUE #257: Make new `dtype_coercion`='stringify' option
This is just an alternative approach to dealing with the issue outlined [here](https://github.com/ToucanToco/fastexcel/issues/256)

The idea being that any type _can_ be casted to a string this would allow a file to be opened without raising on the first try without too much hackiness.

I'm not really sure which is "better" or more desirable.

### PR DESCRIPTION
**Pull request by PrettyWood**
to merge after #298

closes #257

Now you can load a sheet with `dtypes = "string"` directly to set one dtype for all columns

### CODE REVIEW

File: python/tests/test_dtypes.py [hunk 1]
```diff
@@ -257,3 +257,9 @@ def test_dtype_coercion_behavior__strict_sampling_limit(eager: bool) -> None:
     assert pl_df["Mixed dates"].to_list() == [datetime(2023, 7, 21)] * 6 + [None] * 3
     assert pl_df["Asset ID"].dtype == pl.Float64
     assert pl_df["Asset ID"].to_list() == [84444.0] * 7 + [None] * 2
+
+
+def test_one_dtype_for_all() -> None:
+    excel_reader = fastexcel.read_excel(path_for_fixture("fixture-multi-dtypes-columns.xlsx"))
+    sheet = excel_reader.load_sheet(0, dtypes="string")
+    assert sheet.to_polars().dtypes == [pl.String] * 7
```
Comment (lukapeschke) [inline python/tests/test_dtypes.py:265]: could you please also add a check on the sheets `available_columns` to ensure the dtypes contain the right information ?

**Review by lukapeschke — COMMENTED** [review]
LGTM apart from the one missing test, thanks!

File: python/tests/test_dtypes.py (hunk 1 above)
Comment (PrettyWood) [reply python/tests/test_dtypes.py:316]: https://github.com/ToucanToco/fastexcel/pull/299/commits/d9078607d183ccd9578d6307551e03307d995032

**Comment by severinh** [comment]
Thanks a lot for this!!

**Pull request merged**